### PR TITLE
Pushing rubocop.yml

### PR DESCRIPTION
**Pull request by PatrickLab**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  EnforcedStyle: always
+
+Metrics/BlockLength:
+  Exclude:
+    - 'Rakefile'
+    - '**/*.rake'
+    - 'spec/**/*.rb'
+
+Metrics/LineLength:
+  Max: 120
```
Comment (naerymdan) [inline .rubocop.yml]: Why 120 characters for lines, isn't 80 enugh?

**Comment by naerymdan** [comment]
Nice addition!

**Review by naerymdan — CHANGES_REQUESTED** [review]
You need to change that number ASAP.

**Comment by PatrickLab** [comment]
Thanks!

File: .rubocop.yml (hunk 1 above)
Comment (PatrickLab) [reply .rubocop.yml]: Done

**Review by naerymdan — APPROVED** [review]
I approve of those changes.

**Pull request merged**